File: Task_2/task_2.py
def is_pangram():
    sentence = input("Введите строку: ")
    sentence = sentence.lower()
    letters = set()
    for char in sentence:
        if 'a' <= char <= 'z':
            letters.add(char)
    return len(letters) == 26

if __name__ == "__main__":
    if is_pangram():
        print("true")
    else:
        print("false")

File: Task_2/task_2.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout,
});

function isPangram(callback) {
    readline.question('Введите строку: ', (sentence) => {
        const letters = new Set();
        const lowerSentence = sentence.toLowerCase();
        for (const char of lowerSentence) {
            if (char >= 'a' && char <= 'z')
                letters.add(char);
        }
        callback(letters.size === 26);
        readline.close();
    });
}

isPangram((result) => {
    if (result) {
        console.log('true');
    } else {
        console.log('false');
    }
});
